File: Exemplo_C++/ex.cpp
#include "ex.h"
#include <string>
#include <iostream>
using namespace std;


MyClass::MyClass(){}

MyClass::~MyClass(){}

void MyClass::setVal(int v){
	this->val = v;
}

int MyClass::getVal(){ return this->val; }

void MyClass::setNome(string s){
	this->nome = s;
}

string MyClass::getNome(){ return this->nome; }


int main(){

	MyClass A;
	A.setNome("ola mundo!");
	cout << A.getNome() << endl;
}

File: Exemplo_Python/ex.py
class MyClass():
	def __init__(self):
		self._nome = ""
	def setNome(self, nome):
		self._nome = nome;
	def getNome(self):
		return self._nome


def main():
	x = MyClass()
	x.setNome("olar mundo!")
	print(x.getNome())


if __name__ == "__main__":
	main()